eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -276,6 +276,7 @@ stages:
 
     - job: PublishDocsToNightlyBranch
       condition: or(eq(variables['SetDevVersion'], 'true'), and(eq(variables['SetDevVersion'], ''), eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal')))
```

Comment:
Not sure if you wanted to normalize this condition while you are in here as well.

Reply:
There might be situations where we want that https://github.com/Azure/azure-sdk-for-js/blame/e86e2125c1d09a99d42744154ef8fc13c34eae7d/eng/pipelines/templates/stages/archetype-js-release.yml#L235 ... Let's see if that's the case.

Reply:
Indeed some conversation and more research and this no longer applies here. I'll set these conditions to match those in our other repos.